### DISCUSSION TITLE
GPL licensing and static linking

### DISCUSSION THREAD

**Discussion by KyleCardoza** [rsta2/circle#510] [General]
Just a query regarding the intent of the GPL license as it applies to Circle. The GPL considers statically-linked binaries to be derivative works, and thus requires those binaries to be licensed under the same terms if distributed. I just wanted to clarify if you had intended that, or if you would be willing to grant an exemption to cover static linking, which would allow projects that use, but do not modify, Circle to distribute binaries under whatever license is appropriate.

**Comment by rsta2** [comment]
Circle is using source code in important parts, which has been ported from 
other GPL licensed projects like the Linux kernel. Therefore it must use the 
GPL and projects, which are statically linked with Circle have to do this too. 
That is the situation and unfortunately it does not play a role, what my own 
wish is.
